src/bootstrap/src/core/build_steps/format.rs

```diff
@@ -336,7 +336,10 @@ pub fn format(build: &Builder<'_>, check: bool, all: bool, paths: &[PathBuf]) {
         crate::exit!(1);
     }
 
-    if !check {
-        update_rustfmt_version(build);
-    }
+    // Update `build/.rustfmt-stamp`, allowing this code to ignore files which has not been changed
+    // since last merge. This is similar to what `x fmt` does.
```

Comment:
This is a bit misleading, because this function is actually called from `x fmt`.

Reply:
Oh, I did not realize that! I just removed the sentence about `x fmt`.